src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -363,6 +370,33 @@ const LandingPage = () => {
                   {MEDICAL_RECORDS_SETTINGS_LABEL}
                 </Link>
               </section>
+              {!SMHDL && (
```

Comment:
We want to show the link if the feature flag is enabled.

Reply:
fixed